mdc_decode.h

```diff
@@ -61,7 +61,9 @@ typedef void (*mdc_decoder_callback_t)(	int frameCount, // 1 or 2 - if 2 then ex
 										unsigned char extra1,
 										unsigned char extra2,
 										unsigned char extra3,
-										void *context);
+										void *context,
+                    u_int32_t timestamp_absolute,
+                    u_int32_t timestamp_relative);
```

Comment:
I would have used the callback context argument to avoid having to modify the MDC decoder functions; they don't really need to be aware of the timestamps. See the last parameter in `mdc_decoder_set_callback()` - that's a user supplied pointer that gets sent back in the callback function call.